packages/core/src/components/Accordion.tsx

```diff
@@ -0,0 +1,157 @@
+import * as React from "react";
+import {
+  View,
+  StyleSheet,
+  StyleProp,
+  TextStyle,
+  Pressable,
+} from "react-native";
+import Text from "./Text";
+import { withTheme } from "../theming";
+import type { IconSlot } from "../interfaces/Icon";
+import type { Theme } from "../styles/DefaultTheme";
+import { colorTypes } from "@draftbit/types";
+
+import {
+  COMPONENT_TYPES,
+  createTextProp,
+  createNumberProp,
+  createIconProp,
+  createColorProp,
+} from "@draftbit/types";
+
+type Props = {
+  openColor: colorTypes;
+  closedCloser: colorTypes;
+  caretColor: colorTypes;
+  icon?: string;
+  iconSize: number;
+  style: StyleProp<TextStyle>;
+  children: React.ReactNode;
+  title: string;
+  theme: Theme;
+} & IconSlot;
+
+const Accordion = ({
+  Icon,
+  openColor,
+  closedCloser,
+  //caretColor,
+  icon,
+  iconSize = 24,
+  style,
+  title,
+  children,
+  theme: { colors },
+}: Props) => {
+  const [expanded, setExpanded] = React.useState(false);
+
+  const handlePressAction = () => {
+    setExpanded(!expanded);
+  };
+
+  const titleColor = expanded ? colors[openColor] : colors[closedCloser];
+
+  return (
+    <View>
+      <Pressable
+        style={[styles.container, style]}
+        onPress={handlePressAction}
+        accessibilityRole="button"
+      >
+        <View style={styles.row} pointerEvents="none">
+          {icon ? <Icon name={icon} size={iconSize} /> : null}
+          <View style={[styles.item, styles.content]}>
+            <Text
+              selectable={false}
+              style={[
+                styles.title,
+                {
+                  color: titleColor,
+                },
+                style,
+              ]}
+            >
+              {title}
+            </Text>
+          </View>
+          <View style={[styles.item]}>
+            <Icon
+              name={expanded ? "keyboard-arrow-up" : "keyboard-arrow-down"}
```

Comment:
```suggestion
              name={expanded ? "MaterialIcons/keyboard-arrow-up" : "MaterialIcons/keyboard-arrow-down"}
```

icons need to be fully qualified so the underliyng library knows how to pick em

Reply:
done